setup.py

```diff
@@ -610,7 +613,6 @@ def configure(self, ext):
         ext.include_dirs = exclude_from_list(
             ext.include_dirs + self.include_dirs, self.excluded_include_dirs
```

Comment:
This is the only line change that I didn't make from python-igraph's version.

The addition of ext.include_dirs + here does not appear to have come from their setup.py, so I've left it as in the leidenalg version.